contracts/payloads/IGP44/PayloadIGP44.sol

```diff
@@ -0,0 +1,836 @@
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Fluid and Lite
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(targets, values, signatures, calldatas, description);
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Set supply and borrow limit for Dexes on Liquidity Layer
+        action1();
+
+        // Action 2: Set Vault limits on liquidity layer and remove team multisig as auth on vaultFactory
+        action2();
+
+        // Action 3: Update Liquidity User Module Implementation
+        action3();
+
+        // Action 4: Adjust market rates for wstETH, WBTC & cbBTC
+        action4();
+
+        // Action 5: Update iETHv2 Lite Vault to Support wstETH-ETH dex
+        action5();
+    }
+
+    function verifyProposal() external view {}
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    /// @notice Action 1: Set supply and borrow limit for Dexes on Liquidity Layer
+    function action1() internal {
+        {
+            Dex memory DEX_wstETH_ETH = Dex({
+                dex: getDexAddress(1),
+                tokenA: wstETH_ADDRESS,
+                tokenB: ETH_ADDRESS,
+                smartCollateral: true,
+                smartDebt: true,
+                baseWithdrawalLimitInUSD: 12_000_000, // $12M
+                baseBorrowLimitInUSD: 12_000_000, // $12M
+                maxBorrowLimitInUSD: 25_000_000 // $25M
+            });
+            setDexLimits(DEX_wstETH_ETH); // Smart Collateral & Smart Debt
+        }
+
+        {
+            Dex memory DEX_USDC_USDT = Dex({
+                dex: getDexAddress(2),
+                tokenA: USDC_ADDRESS,
+                tokenB: USDT_ADDRESS,
+                smartCollateral: false,
+                smartDebt: true,
+                baseWithdrawalLimitInUSD: 0, // $0
+                baseBorrowLimitInUSD: 7_500_000, // $7.5M
+                maxBorrowLimitInUSD: 15_000_000 // $15M
+            });
+            setDexLimits(DEX_USDC_USDT); // Smart Debt
+        }
+
+        {
+            Dex memory DEX_cbBTC_WBTC = Dex({
+                dex: getDexAddress(3),
+                tokenA: cbBTC_ADDRESS,
```

Comment:
not sure if it matters in code but can we make tokenA always token0? that would be token0 = WBTC token1 = CBBTC at the dex

Reply:
@bergben Yess! To avoid confusion using TokenA and TokenB instead of Token0 and Token1 and in the codebase it doesn't matter as of now. But next time, If there is a need we will sort based on it and can use it.